scripts/upgrade_admin_server.sh

```diff
@@ -116,6 +116,13 @@ n.save"
     rm -f /etc/systemd/system/crowbar.service
     systemctl daemon-reload
 
+    # Make sure latest chef data are uploaded
+    # Normally packages should take care of it, but for the rare case of chef-server
+    # not being available during the upgrade, better make sure and do it explicitely.
+    systemctl restart chef-server
+    pushd /opt/dell/crowbar_framework
+    sudo -u crowbar RAILS_ENV=production bin/rake chef:upload:all
```

Comment:
I would be happier if the rake command was the one waiting for the chef server to be available instead of us monkey patching it here.

Not a blocker at all, just a question and a note for any bugs that may arise from this :)